Formula/pysvn.rb

```diff
@@ -0,0 +1,70 @@
+class Pysvn < Formula
+  desc "Python interface to Subversion"
+  homepage "http://pysvn.tigris.org/"
+  url "http://pysvn.barrys-emacs.org/source_kits/pysvn-1.8.0.tar.gz"
+  sha256 "39596f4884ed689cdb5a4e210e421724302a566c7ba756cc4d46bbfeb0c8326b"
+
+  depends_on "openssl"
+  depends_on "subversion"
+  depends_on :python => :recommended
+  depends_on :python3 => :recommended
+
+  # Fix configure to respect CPPFLAGS, CXXFLAGS and LDFLAGS
+  patch :DATA
+
+  def install
+    if build.without?("python") && build.without?("python3")
+      odie "pysvn: at least one of --with-python or --with-python3 must be specified"
+    end
+    cd "Source" do
+      inreplace "setup_configure.py", "%(PYTHON_FRAMEWORK)s", "-undefined dynamic_lookup"
+      Language::Python.each_python(build) do |python, version|
+        system python, "setup.py", "configure"
+        system "make"
+        (lib/"python#{version}/site-packages/pysvn").install "pysvn/__init__.py", Dir["pysvn/_pysvn*.so"]
+        system "make", "clean"
```

Comment:
Because we need to start from scratch (reconfigure & recompile) for every new Python in cycle (perhaps `make clean` can be moved before running `make` and not after install).